src/ContractInteractor.ts

```diff
@@ -594,33 +594,23 @@ export default class ContractInteractor {
             .encodeABI();
     }
 
-    async getActiveRelays(
+    async getActiveRelayInfo(
         relayManagers: Set<string> | string[]
-    ): Promise<RelayData[]> {
-        const managers: string[] = Array.from(relayManagers);
-        const contractCalls: Array<Promise<RelayData>> = [];
-        managers.forEach((managerAddress) => {
-            contractCalls.push(
-                this.relayHubInstance.getRelayData(managerAddress)
-            );
-        });
-        const results = await Promise.all(contractCalls);
+    ): Promise<RelayManagerData[]> {
+        const results = await this.getRelayInfo(relayManagers)
         return results.filter(
-            (relayData) =>
-                !relayData.penalized &&
-                relayData.stakeAdded &&
-                relayData.registered
+            (relayData) => relayData.registered && relayData.currentlyStaked
```

Comment:
the now removed `penalized` is covered by `currentlyStaked` (which also replaces `stakeAdded`)